content/4_Gold/LIS.mdx

```diff
@@ -132,8 +140,11 @@ int find_lis(vector<int> a) {
 	vector<int> dp;
 	for (int i : a) {
 		int pos = lower_bound(dp.begin(), dp.end(), i) - dp.begin();
-		if (pos == dp.size()) dp.push_back(i);
-		else dp[pos] = i;
+		if (pos == dp.size()) {  // we can have a new, longer IS!
+			dp.push_back(i);
+		} else {  // oh ok, at least we can make the ending element smaller
+			dp[pos] = i;
+		}
```

Comment:
Fine as is, no?

Reply:
I feel like just adding braces make it look better. I can still remove them if you want.

Reply:
Oh, I personally think braces for 1-liners looks messy. Generally I leave stuff like this to the author of the code though

Reply:
My two cents: leave the braces in so control flow is easier to understand.